app/assets/javascripts/components/archive/ArchiveApp.es6.jsx

```diff
@@ -4,8 +4,17 @@ class ArchiveApp extends React.Component {
   }
 
   render() {
+  	var videoComponents = this.props.data.map(function(video) {
+	    return (
+	       <ArchiveVideoCard 
+	          video_id={video["IDv2"]} 
+	          video_title='Edith speaking Spanish' 
```

Comment:
Any way we can generate a title from the metadata?

Reply:
I looked into it. There's not actually a title field. For now, I think we leave it like this and have a TODO to generate the title.

Reply:
sounds good. If there's an easy way to do it, we can get this done by Friday.